scripts/release/mule/sign/sign.sh

```diff
@@ -17,19 +16,12 @@ PKG_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
 SIGNING_KEY_ADDR=dev@algorand.com
 STATUSFILE="build_status_${CHANNEL}_${VERSION}"
 
+mkdir -p "$PKG_DIR"
 cd "$PKG_DIR"
 
 if [ -n "$S3_SOURCE" ]
 then
-    PREFIX="$S3_SOURCE/$CHANNEL/$VERSION"
-
-    # deb
-    aws s3 cp "s3://$PREFIX/algorand_${CHANNEL}_${OS_TYPE}-${ARCH_TYPE}_${VERSION}.deb" .
-    aws s3 cp "s3://$PREFIX/algorand-devtools_${CHANNEL}_${OS_TYPE}-${ARCH_TYPE}_${VERSION}.deb" .
-
-    # rpm
-    aws s3 cp "s3://$PREFIX/algorand-${VERSION}-1.${ARCH_BIT}.rpm" .
-    aws s3 cp "s3://$PREFIX/algorand-devtools-${VERSION}-1.${ARCH_BIT}.rpm" .
+    aws s3 sync "s3://$S3_SOURCE/$CHANNEL/$VERSION/$OS_TYPE/$ARCH_TYPE/" .
```

Comment:
It looks like you were using what's return by uname before, do you know if my update to use the arch common name rather than the technical name might cause problems on installs?

Reply:
Do you mean installing using yum?  It shouldn't matter, what matters is the information in the spec file.

Reply:
oh ok cool